website/static/js/filepage/index.js

```diff
@@ -161,41 +161,22 @@ var FileViewPage = {
         //This code was abstracted into a panel toggler at one point
         //it was removed and shoved here due to issues with mithrils caching and interacting
         //With other non-mithril components on the page
-        var panels;
+        ctrl.enableEditing();
+        var panels, shown;
         if (ctrl.editor) {
-            panels = [ctrl.editor, ctrl.revisions];
+            panels = [ctrl.editor];
         } else {
-            panels = [ctrl.revisions];
+            panels = [];
         }
```

Comment:
Although this whole panels thing is no longer necessary, since edit would be the only possible one, I like it as a way to not need to check for editor later, in the panels.map functions.